documentation/rfc/line_sections.md

```diff
@@ -0,0 +1,165 @@
+
+## pt ref
+
+I think an example will be easier to understand. We impact the stops [C, D, E] of the 'route_1_of_the_line_1'
+
+```
+A           B           C           D           E           F
+o --------- o --------- o --------- o --------- o --------- o   <- route_1_of_the_line_1
+
+                        XXXXXXXXXXXXXXXXXXXXXXXXX               <- impact on the route_1_of_the_line_1 on [C, E]
+```
+
+### common cases
+
+I think in a v1 (but I don't think we'll do a v2 :wink: ) we can keep it quite simple (but not that coherent :confused: ).
+
+On ptref we can display the line section disruption on a ptref call if the last object of the ptref call is directly linked to an **active** disruption
+
+examples:
+
+this is the navitia call and YES means we display the disruption, NO means we do not display it
+
+`/routes/route_1_of_the_line_1` --> YES (? not sure about this one if the line section is represented as linked to the line)
+
+`/stop_areas/A`  --> NO
+`/stop_areas/B`  --> NO
+`/stop_areas/C`  --> YES
+`/stop_areas/D`  --> YES
+`/stop_areas/E`  --> YES
+`/stop_areas/F`  --> NO
+
+`/routes/route_2_of_the_line_1`  --> NO
+
+`/lines/line_1`  -->  YES (? not sure about this one if we represent the line section as linked to 2 stops and a list of routes)
+`/lines/line_2`  -->  NO
+
+If we decide to display the line section disruptions based on the last object of the query it will lead to some difficult to understand behaviour:
+
+`/stop_areas/A/routes/route_1_of_the_line_1`  --> YES
+`/stop_areas/C/routes/route_2_of_the_line_1`  --> NO
+
+`/networks/network_of_the_route_1 `  --> NO
+
+`/stop_points/C_1`  --> NO
+`/vehicle_journeys/vj1_of_route_1_of_line_1`  --> NO
+
+
+### ptref with /disruptions
+
+This is for the calls api.navitia.io/v1/coverage/bob/<some pt ref filters>/disruptions
+
+The /disruption api is meant to be a direct representation of the object model below so I think it's ok the keep the same mechanism as the common cases (but not restricted ot the active disruptions by default)
+
+### ptref with /traffic_reports
+
+This is for the calls api.navitia.io/v1/coverage/bob/<some pt ref filters>/traffic_reports
+
+This /traffic_reports api is meant to be a clever representation of the disruptions active on the object and thus we can do more things on this
+
+`/stop_areas/B/traffic_reports`  --> NO
+`/stop_areas/C/traffic_reports`  --> YES
+
+`/lines/line_1/traffic_reports ` -->  YES
+`/lines/line_2/traffic_reports`  -->  NO
+
+`/routes/route_1_of_the_line_1/traffic_reports`  --> YES
+`/routes/route_2_of_the_line_1/traffic_reports`  --> NO
+
+`/stop_areas/A/routes/route_1_of_the_line_1/traffic_reports`  --> YES (I think NO would be better it but this will be quite difficult for not that much gain)
+`/stop_areas/C/routes/route_2_of_the_line_1/traffic_reports`  --> NO
+
+`/networks/network_of_the_route_1/traffic_reports `  --> YES
+`/stop_points/C_1/traffic_reports`  --> YES since it's stop_area is part of an impacted line section
```

Comment:
you mix sp and sa, I think it's worth describe both (and I'd say YES for saC, spC1 and NO for spC2)